WinUIGallery/Samples/SamplePages/SampleWindow1.xaml.cs

```diff
@@ -30,10 +30,26 @@ public SampleWindow1(string WindowTitle, Int32 Width, Int32 Height, Int32 X, Int
         appWindow.Resize(new Windows.Graphics.SizeInt32(Width, Height));
 
         // Set the window position on screen
-        appWindow.Move(new Windows.Graphics.PointInt32(X, Y));
-
-        // Set the window icon
-        appWindow.SetIcon("Assets/Tiles/GalleryIcon.ico");
+        appWindow.Move(new Windows.Graphics.PointInt32(X, Y));
+
+        // Set the preferred theme for the title bar
+        appWindow.TitleBar.PreferredTheme = TitleBarPreferredTheme switch

```

Comment:
@Zakariathr22
this can be simplified to this:

appWindow.TitleBar.PreferredTheme = EnumHelper.GetEnum<TitleBarTheme>(TitleBarPreferredTheme);